NGitLab/Impl/Json/BooleanConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Impl.Json
+{
+    internal sealed class BooleanConverter : JsonConverter<bool>
```

Comment:
Could you add some tests for these converters?

Reply:
The converters were added to make existing tests pass. So, the converter code is already validated (and without it some tests would fail). Do you still think we need to add more tests?

Reply:
You don't need to validate them all, but I would prefer some tests to validate it correctly handle `null` values and also the parsing of datetime in different format (and that they are all UTC).

Reply:
I added some tests. Let me know if you feel there should be more.